ntbs-service/Services/AzureAdDirectoryService.cs

```diff
@@ -24,30 +22,89 @@ public interface IAzureAdDirectoryService : IDisposable
         /// </returns>
         Task<IEnumerable<(Models.Entities.User user, List<TBService> tbServicesMatchingGroups)>> LookupUsers(IList<TBService> tbServices);
 
-        bool IsGroupAnNtbsGroup(string groupName);
-
         Task<IList<Claim>> BuildRoleClaimsForUser(string userPrincipalName);
     }
 
     public class AzureAdDirectoryService : IAzureAdDirectoryService
     {
         private readonly IGraphServiceClient _graphServiceClient;
-        private readonly AzureAdOptions _adOptions;
+        private readonly AdOptions _adOptions;
 
         public AzureAdDirectoryService() {}
 
-        public AzureAdDirectoryService(IGraphServiceClient graphServiceClient, AzureAdOptions adOptions)
+        public AzureAdDirectoryService(IGraphServiceClient graphServiceClient, AdOptions adOptions)
         {
-            this._graphServiceClient = graphServiceClient;
-            this._adOptions = adOptions;
+            _graphServiceClient = graphServiceClient;
+            _adOptions = adOptions;
         }
 
         public void Dispose()
         {
-            
+
+        }
+
+        public async Task<IEnumerable<(Models.Entities.User user, List<TBService> tbServicesMatchingGroups)>> LookupUsers(IList<TBService> tbServices)
```

Comment:
Functionally, I haven't really changed anything in this class, but I noticed that several methods were public that didn't need to be so I've done some refactoring.